src/layout/Header.jsx

```diff
@@ -1,7 +1,84 @@
 import React from "react";
+import { Link as GatsbyLink, useStaticQuery, graphql } from "gatsby";
+import { makeStyles } from "@material-ui/core/styles";
+import Toolbar from "@material-ui/core/Toolbar";
+import IconButton from "@material-ui/core/IconButton";
+import SearchIcon from "@material-ui/icons/Search";
+import Typography from "@material-ui/core/Typography";
+import Link from "@material-ui/core/Link";
 
-const Header = () => {
-  return <>Header</>;
+const useStyles = makeStyles((theme) => ({
+  toolbar: {},
+  toolbarTitle: {
+    flex: 1,
```

Comment:
The MUI `Box` component has flexbox attributes built in. I'm not sure if it is "correct" to use it that way vs using `makeStyles`, but it might be worth looking into if the classes are not reused.

Reply:
I think both ways are valid. MUI seems to use only the makestyles way for its appbar doc.